src/ConnectedMode.UnitTests/ProjectRootCalculatorTests.cs

```diff
@@ -49,7 +49,7 @@ public void MefCtor_CheckIsSingleton()
     public async Task CalculateBasedOnLocalPathAsync_StandaloneMode_ReturnsNull()
     {
         var testSubject = CreateTestSubject(out _, out var configurationProviderMock, out _);
```

Comment:
Change the variable name maybe

Reply:
Well, since this is pretty much is used like a configuration provider, I think it's not too confusing, and I'd rather just leave it like this and be done with the pr